### PR TITLE
raylib: update checksum

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
See https://github.com/raysan5/raylib/issues/2942 for verification.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Confirmed upstream

**Comment by chenrui333** [comment]
Looks like build failed.

File: Formula/raylib.rb [hunk 1]
```diff
@@ -28,20 +28,22 @@ class Raylib < Formula
...
+                               *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    system "cmake", "-S", ".", "-B", "build-static",
+                               "-DBUILD_SHARED_LIBS=OFF",
+                               "-DMACOS_FATLIB=OFF",
+                               "-DBUILD_EXAMPLES=OFF",
+                               "-DBUILD_GAMES=OFF",
+                               *std_cmake_args
+    system "cmake", "--build", "build-static"
```
Comment (ZhongRuoyu) [inline Formula/raylib.rb:45]: Changed `build` to `build-static`. It should work now.

**Review by ZhongRuoyu — APPROVED** [review]
The only failure is below, which is expected. Merging now.

```
==> brew audit raylib --online --git --skip-style
==> FAILED
Full audit raylib --online --git --skip-style output
  raylib:
    * stable sha256 changed without the url/version also changing; please create an issue upstream to rule out malicious circumstances and to find out why the file changed.
  Error: 1 problem in 1 formula detected
```

**Pull request closed without merging**